docs/reference/filebeat/filebeat-input-filestream.md

```diff
@@ -6,17 +6,24 @@ mapped_pages:
 
 # filestream input [filebeat-input-filestream]
 
+::::{important}
+Filestream, with default settings will only start ingesting files **when
+their size is >= 1024 bytes**. To change this behaviour you can
+configure a different length for the fingerprint by setting
+[`prospector.scanner.fingerprint.length`](#filebeat-input-filestream-scan-fingerprint).
```

Comment:
```suggestion
By default, Filestream will only start ingesting files **when
their size is >= 1024 bytes**. To change this behaviour you can
configure a different length for the fingerprint by setting
[`prospector.scanner.fingerprint.length`](#filebeat-input-filestream-scan-fingerprint).
```

Reply:
We also generally use American English in docs and in [UI text](https://eui.elastic.co/docs/content/language/) for consistency (so `behavior` instead of `behaviour`), but it looks like we already use `behaviour` elsewhere on this page outside the context of this PR.

Reply:
I learnt British English, and lived in Scotland for over a year, sometimes it is a bit hard to remember which words are different between British and American English....